src/docs/api/greencheck/01_v3/check-single-domain.md

```diff
@@ -0,0 +1,144 @@
+---
+tags: apiMain
+libraryName: Greencheck API
+title: v3 - Check a domain
+description: Use this API to check if a domain is hosted on a green web host recognised by The Green Web Foundation.
+hasTabs: true
+eleventyNavigation:
+  key: v3-check-single-domain
+  title: Check a domain
+  order: 1
+  # sectionTitle: Version 3
+---
+
+# Query for a single domain
+
+The Greencheck API provides endpoints that allow developers to query [The Green Web Foundation's dataset](https://datasets.thegreenwebfoundation.org/) of green domains.
+
+The endpoint allows you to check if a single website domain is served from a green web host.
+
+## Request method
+
+Use the `GET` method for this request.
+
+## Endpoint
+
+The request should be sent to `http://api.thegreenwebfoundation.org/api/v3/greencheck/[hostname]`.
+
+The `[hostname]` parameter should be replaced with the website domain you want to query for.
+
+## Parameters
+
+### `[hostname]`
+
+A valid hostname should be passed to the endpoint. This parameter must not include any protocol, port, or path information.
+
+- `climateaction.tech` <span class="badge align-middle badge-success">Accepted</span>
+- `https://climateaction.tech` <span class="badge align-middle badge-error">Incorrect</span>
+- `climateaction.tech/events` <span class="badge align-middle badge-error">Incorrect</span>
+
+## Sample request
+
+<seven-minute-tabs>
+   <ol role="tablist" aria-label="Select a programming language to preview">
+    <li><a href="#js" role="tab" aria-selected="true">JavaScript</a></li>
+    <li><a href="#curl" role="tab">cURL</a></li>
+   </ol>
+
+   <div id="js" role="tabpanel">
+{% set code %}
+fetch("http://api.thegreenwebfoundation.org/api/v3/greencheck/climateaction.tech", {
+  method: "GET",
+}).then((response) => response.json());
+{% endset %}
+
+{% codeSnippet code, 'js' %}
+
+   </div>
+
+   <div id="curl" role="tabpanel">
+{% set code %}
+curl -X 'GET' \
+ 'http://api.thegreenwebfoundation.org/api/v3/greencheck/climateaction.tech' \
+ -H 'accept: application/json'
```

Comment:
When I executed this exact command, I got `301 Moved Permanently` instead of the expected JSON. The behavior changed when I added the flag `-L` to follow redirects, [as suggested here](https://stackabuse.com/follow-redirects-in-curl/). I think it would be a good idea to add this flag to the example command

(this applies to curl commands to both API endpoints)

Reply:
Thanks @tortila. It seems to work with a `https` protocol though. Would it be better to guide people towards using the https version of the endpoint?

Reply:
Ah, I didn't spot that - I think we should use the `https` endpoint then